pytorch_lightning/utilities/fetching.py

```diff
@@ -282,23 +288,21 @@ class InterBatchParallelDataFetcher(DataFetcher):
         batch 2:             [HtoD]                          [forward][backward]
     """
 
-    def __init__(self, *args, **kwargs) -> None:
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Just out of curiosity? Why do we go with *args and **kwargs here if we know the signature of `super().__init__`?

Reply:
Just for simplicity, we could add them too and have them in both places